README.md

```diff
@@ -83,6 +83,7 @@ The `containerTemplate` is a template of container that will be added to the pod
 * **command** The command the container will execute.
 * **args** The arugments passed to the command.
 * **ttyEnabled** Flag to mark that tty should be enabled.
+* **ports** Expose ports on the container.
```

Comment:
@jwhitcraft in what scenario are you using it? Because I am trying to run some tests with a mariadb database and getting connection refused when trying to connect. My pod template is exactly as the one on readme. I tried to connect to mysql cli from within the container as well, and I get a ´Can't connect to local MySQL server through socket ´ error (commented in the PR as well).